src/ai/ai.hpp

```diff
@@ -137,8 +137,9 @@ class AI {
 
 	/**
 	 * Get the number of days before the next AI should start.
+	 * @param ids_to_skip Number of company indexes to skip, that are pretended to have been started, but actually aren't yet.
```

Comment:
This sentence is hard to parse. Especially so, because the function is `GetStartNextTime`, so .. it feels a bit weird. It needs more context of some sorts. Possible suggestion (no clue if it is any good):
"For which Nth AI to retrieve the start_date.".
Possibly that needs a rename of the variable too.